website/docs/advanced/predefined-variations.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: predefined-variations
+title: Predefined Variations
+description: Learn about Free-form values and Predefined Variations in ConfigCat, how they differ, and when to use each approach for managing feature flag values.
```

Comment:
As per [section 5.4. of text guidelines](https://github.com/configcat-labs/content-wiki/blob/main/text-guidelines.md#54-technical-terms-and-concepts), we should not capitalize terms other than those listed there.

```suggestion
description: Learn about free-form values and predefined variations in ConfigCat, how they differ, and when to use each approach for managing feature flag values.
```